app/scripts/modules/appengine/instance/details/details.controller.ts

```diff
@@ -69,7 +70,6 @@ class AppengineInstanceDetailsController {
       header: 'Really terminate ' + shortName + '?',
       buttonText: 'Terminate ' + shortName,
       account: instance.account,
-      provider: 'appengine',
```

Comment:
@danielpeach I did not see anywhere that would use this field - can you confirm everything works as expected right now if you remove this field?